search/indico_search/templates/results.html

```diff
@@ -24,7 +24,7 @@
                     {% block banner %}{% endblock %}
                 </div>
 
-                {{ form_header(form, i_form=false) }}
+                {{ form_header(form,method = 'get', i_form=false) }}
```

Comment:
missing space after `,` and there should not be spaces around `=` for keyword arguments